cpp/src/io/utilities/column_buffer.hpp

```diff
@@ -140,8 +142,7 @@ struct column_buffer {
 std::unique_ptr<column> make_column(column_buffer& buffer,
                                     column_name_info* schema_info,
                                     std::optional<reader_column_schema> const& schema,
-                                    rmm::cuda_stream_view stream,
-                                    rmm::mr::device_memory_resource* mr);
+                                    rmm::cuda_stream_view stream);
```

Comment:
Not really matter but the doc for `mr` can be removed.

Reply:
Done.